### PR TITLE
Wrangler specify identifiers

### PR DESCRIPTION
**Pull request by daryanypl**

### CODE REVIEW

File: axis.py [hunk 1]
```diff
@@ -227,12 +227,12 @@ class Axis360BibliographicCoverageProvider(BibliographicCoverageProvider):
...
-    def __init__(self, _db, input_identifier_types=None, 
-                 metadata_replacement_policy=None, axis_360_api=None,
-                 **kwargs):
-        # We ignore the value of input_identifier_types, but it's
-        # passed in by RunCoverageProviderScript, so we accept it as
-        # part of the signature.
+    def __init__(self, _db, metadata_replacement_policy=None, axis_360_api=None,
+                 input_identifier_types=None, input_identifiers=None, **kwargs):
+        """
+        :param input_identifier_types Passed in by RunCoverageProviderScript, data sources to get coverage for.
```
Comment (courte) [inline axis.py]: Add colon.

File: coverage.py [hunk 2]
```diff
@@ -245,10 +245,15 @@ def process_batch_and_handle_results(self, batch):
...
-                        "Transient failure covering %r: %s", 
-                        item.obj, item.exception
-                    )
+                    try:
+                        self.log.warn(
+                            "Transient failure covering %r: %s", 
+                            item.obj, item.exception
+                        )
+                    except Error, e:
+                        set_trace()
```
Comment (courte) [inline coverage.py]: Remove.

File: scripts.py [hunk 3]
```diff
@@ -206,6 +206,7 @@ def look_up_identifiers(cls, _db, parsed, stdin_identifier_strings, *args, **kwa
         """Turn identifiers as specified on the command line into
         real database Identifier objects.
         """
+        #set_trace()
```
Comment (courte) [inline scripts.py]: Remove (and below).

File: scripts.py [hunk 4]
```diff
@@ -403,25 +405,33 @@ def parse_command_line(cls, _db, cmd_args=None, stdin=sys.stdin,
...
-                cutoff_time=args.cutoff_time,
+                cutoff_time=parsed_args.cutoff_time,
                 **kwargs
             )
         self.provider = provider
         self.name = self.provider.service_name
-        self.identifiers = args.identifiers
+
 
     def extract_additional_command_line_arguments(self, args):
```
Comment (courte) [inline scripts.py]: Remove `args` param.

File: axis.py (hunk 1 above)
Comment (daryanypl) [inline axis.py]: yep

File: coverage.py (hunk 2 above)
Comment (daryanypl) [inline coverage.py]: done

File: scripts.py (hunk 4 above)
Comment (daryanypl) [inline scripts.py]: done

**Review by courte — APPROVED** [review]
👍 🌈

**Pull request merged**